### PR TITLE
[FIX] hr: fix TypeError: '>' not supported between instances of 'bool' and 'datetime.datetime'

### PR DESCRIPTION
**Pull request by micheledic**
### Before This PR

if there the new_hire_field is False(in case of hr_contract for example if there are not contracts), an error appear.
### After THIS PR
 This Commit will check  that the field is not false so the error is not raised 

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr/models/hr_employee_base.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def _compute_newly_hired(self):
         new_hire_field = self._get_new_hire_field()
         new_hire_date = fields.Datetime.now() - timedelta(days=90)
         for employee in self:
-            employee.newly_hired = employee[new_hire_field] > new_hire_date
+            employee.newly_hired = employee[new_hire_field] > new_hire_date if isinstance(employee[new_hire_field], datetime) else False
```
Comment (tivisse) [inline addons/hr/models/hr_employee_base.py:77]: We could manage the case when this is a date and not a datetime (eg when the field ``first_contract_date`` is used)
```suggestion
            if not employee[new_hire_field]:
                employee.newly_hired = False
            elif not isinstance(employee[new_hire_field], datetime):
                employee.newly_hired = employee[new_hire_field] > new_hire_date.date()
            else:
                employee.newly_hired = employee[new_hire_field] > new_hire_date
```

**Comment by tivisse** [comment]
Hi @micheledic 

Thanks for your feedback !

I've made a suggestion, can you adapt your code accordingly ? 

Thanks ! :)

**Comment by micheledic** [comment]
> Hi @micheledic
> 
> Thanks for your feedback !
> 
> I've made a suggestion, can you adapt your code accordingly ?
> 
> Thanks ! :)

done!

**Comment by tivisse** [comment]
@robodoo r+

Thanks for your feedback :)

**Pull request closed without merging**